e2e/vm-sidebar.e2e-spec.ts

```diff
@@ -21,12 +21,12 @@ describe('e2e-test-vm-sidebar', () => {
     login.navigateTo('/');
     login.login();
     login.waitRedirect('instances');
+    sidebar = new VMSidebar();
   });
 
   beforeEach(() => {
     vmlist = new VMList();
     deploy = new VMDeploy();
-    sidebar = new VMSidebar();
   });
 
   /* it('Verify VM color is changed', () => {
```

Comment:
You forgot to uncomment your tests. =) 
Uncomment and make sure your tests run on the latest version of master.